core/network/src/protocol/sync.rs

```diff
@@ -837,7 +886,7 @@ impl<B: BlockT> ChainSync<B> {
 		};
 		let is_required_data_available =
 			!requires_additional_data &&
-			range.end - range.start == One::one() &&
+			 range.end - range.start == One::one() &&
```

Comment:
```suggestion
			range.end - range.start == One::one() &&
```

Reply:
Not sure how this ended up here actually xD